### DISCUSSION TITLE
Everything synced and all taks in Ticktick got updated with default folder location

### DISCUSSION THREAD

**Discussion by aagha** [thesamim/TickTickSync#162] [Q&A] [answered]
First off, very excited to learn about this project.

I made a ticktick backup before starting.

Note, I have "Full vault sync" turned off, but it seems that ALL my ticktick's sync'd:

![image](https://github.com/user-attachments/assets/bf27c7bf-cf04-487a-bf4b-7d0c9391e23b)

As a result, in tictick, all my tasks now have a #ticktick tag and contain the location of the task in Obsidian--this is unexpected (to me):

![image](https://github.com/user-attachments/assets/9e00239f-65dc-4bc6-b42b-4d0735eb1dbd)

In the same vein, I have created a task using the right tag in my Daily notes, that did NOT sync to TickTick:

![image](https://github.com/user-attachments/assets/7412b28d-d457-49c7-a9b6-5956e95f0b71) (note the "USPD" typo)

![image](https://github.com/user-attachments/assets/96cfe963-711e-4331-b8f9-5c5d6f607a6c)

Did I configure something incorrectly?

**Comment by thesamim** [comment, marked as answer]
@aagha : Pardon the delayed response.

"Full Vault Sync" only affects the Obsidian to TickTick synchronization. Meaning that it will sync ALL tasks in the Obsidian vault whether they are tagged or not. 

If you want to limit the tasks synched FROM TickTick to Obsidian, please fill in the "Limit Synchronization" section of TickTickSync settings to limit by Project (List in TickTick) or Tag or Both.

| In the same vein, I have created a task using the right tag in my Daily notes, that did NOT sync to TickTick:
I'm **assuming** that by right tag you mean #ticktick. If that's the case I don't understand why it wouldn't have synched. It should have gone to your Inbox. If you are able to reproduce that, please let me know.
